optout/src/main/java/gov/cms/ab2d/optout/OptOutConstants.java

```diff
@@ -0,0 +1,28 @@
+package gov.cms.ab2d.optout;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.endpoints.internal.Value;
+
+public class OptOutConstants {
+    public static final String BFD_S3_BUCKET_NAME = "ab2d-opt-out-temp-349849222861-us-east-1";
+    public static final Region S3_REGION = Region.US_EAST_1;
+    public static final int DEFAULT_LINE_LENGTH = 459;
+    public static final int MBI_INDEX_START = 0;
+    public static final int MBI_INDEX_END = 11;
```

Comment:
Do we want to have these constants stored somewhere in secret manager  in V2? or so if they change the length it would be easy to modify and don't have to go through deployments? thoughts?

Reply:
Interesting... Thanks for the idea! I will think about it